ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install.sh

```diff
@@ -44,6 +44,10 @@ $COMMON_DIR/copy_test_file.sh
 # diable auto kernel updates
 $UBUNTU_COMMON_DIR/disable_auto_upgrade.sh
 
+cp $UBUNTU_COMMON_DIR/check_nvidia_uvm.sh .
```

Comment:
Is this a problem also for the NC SKUs? If not, we can skip this change for LTS image.

Reply:
It appears to be a problem at least for Standard_NC64as_T4_v3, any other SKUs it would be better for me to check?

Reply:
Is this alright @jithinjosepkl ?

Reply:
Can we pls check for the NC24r VM as well? (targeted SKU for LTS image).